<launch>
  <!-- Define arguments with default values -->
  <arg name="target_x" default="5.0" />
  <arg name="target_y" default="5.0" />

  <!-- Launch the turtlesim node -->
  <node name="turtlesim" pkg="turtlesim" type="turtlesim_node" />

  <!-- Launch the go_to node with remapping and arguments -->
  <node name="go_to" pkg="go_to" type="go_to" output="screen">
    <remap from="/cmd_vel" to="/turtle1/cmd_vel" />
    <param name="target_x" value="$(arg target_x)" />
    <param name="target_y" value="$(arg target_y)" />
  </node>
</launch>

